lib/BaseController.js

```diff
@@ -556,9 +566,6 @@ module.exports = class BaseController {
 
         let original = clone(file);
         this.reconcileFields(file, lastApplied);
```

Comment:
```suggestion
        this.reconcileFields(file, lastApplied);
        if (objectPath.get(file, ['metadata', 'annotations']) === null) {
          objectPath.set(file, ['metadata', 'annotations'], {});
        }
```

Reply:
sorry.. i had a push issue... i had done these things

Reply:
ah i had wondered why you ignored me 😄